templates/toss/src/lib/wallet.ts

```
@@ -0,0 +1,255 @@
+      hex = hex.slice(2);
+    }
+    const bytes = new Uint8Array(hex.length / 2);
+    for (let i = 0; i < hex.length; i += 2) {
+      bytes[i / 2] = parseInt(hex.substr(i, 2), 16);
+    }
+    return bytes;
+  }
+
+  static decrypt(encryptedData: string, stringKey: string): any {
+    const key = keccak256(toHex(stringKey.toLowerCase()));
+    const encrypted = this.hexToBytes(encryptedData);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)